extensions/amp-analytics/0.1/vendors.js

```diff
@@ -2105,5 +2109,20 @@ ANALYTICS_CONFIG['adobeanalytics_nativeConfig']
 ANALYTICS_CONFIG['oewa']['triggers']['pageview']['iframe' +
 /* TEMPORARY EXCEPTION */ 'Ping'] = true;
 
-// Merge ANALYTICS_IFRAME_TRANSPORT_CONFIG into ANALYTICS_CONFIG
-Object.assign(ANALYTICS_CONFIG, ANALYTICS_IFRAME_TRANSPORT_CONFIG);
+mergeIframeTransportConfig(ANALYTICS_CONFIG, IFRAME_TRANSPORTS);
+
+/**
+ * Merges iframe transport config.
+ *
+ * @param {!JsonObject} config
+ * @param {!JsonObject} iframeTransportConfig
+ */
+function mergeIframeTransportConfig(config, iframeTransportConfig) {
+  for (const vendor in iframeTransportConfig) {
+    if (hasOwn(iframeTransportConfig, vendor)) {
+      const url = iframeTransportConfig[vendor];
+      config[vendor]['transport'] =
+          Object.assign({}, config[vendor]['transport'], {'iframe': url});
```

Comment:
Have one question: by combining the `transport` config, those vendors with `transport: iframe` will get 
`transport {beacon: true, xhrpost: true, image:true}` Will a request also be sent through those transport method?

Reply:
Hmm, good point. This is why it was previously:
```

export const ANALYTICS_IFRAME_TRANSPORT_CONFIG = /** @type {!JsonObject} */ ({
  'bg': {
    'transport': {
      'iframe': 'https://tpc.googlesyndication.com/b4a/b4a-runner.html',
    },
  },
```

Reply:
We ignore all the other options if iframe transport is provided. See amp-analytics.js sendRequest.

Reply:
I see. Thanks.